### ISSUE TITLE
Architect

### ISSUE THREAD

**Issue by chbe8475** [Azure/azure-cli#12287]
The different parameters should have a list of accepted values to make the documentation even better. Like in the Storage section: https://docs.microsoft.com/en-us/cli/azure/storage/account?view=azure-cli-latest#az-storage-account-create

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 15cb3fd0-5d13-ab34-223b-f7972aae16a2
* Version Independent ID: 65cc59ca-c366-dbe0-b95d-7dcfa3a4438d
* Content: [az functionapp plan](https://docs.microsoft.com/en-us/cli/azure/functionapp/plan?view=azure-cli-latest#az-functionapp-plan-create)
* Content Source: [src/azure-cli/azure/cli/command_modules/appservice/_help.py](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/appservice/_help.py)
* Service: **functions**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by yonzhan** [comment]
add to S167.

**Comment by anthonychu** [comment]
Thanks @chbe8475. Can you provide an example? Are you thinking about options for available `sku`s?

**Comment by chbe8475** [comment]
Yes skus specifically in this case. But if there are other parameters with
only a fixed set of valid values those should be documented in my opinion.

/Christer

Den fre 28 feb. 2020 kl 19:00 skrev Anthony Chu <notifications@github.com>:
